ui/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "clean": "rm -rf src/generated",
-    "codegen": "npm run clean && java -jar ../tools/swagger-codegen-cli.jar generate --lang typescript-angular  --input-spec ../api/src/main/resources/workbench.yaml  --output src/generated --additional-properties ngVersion=2"
```

Comment:
Oversight. I thought that was doing a full clean which seemed overkill. I renamed for clarity.